pkg/analysis/passes/sdkusage/sdkusage.go

```diff
@@ -3,23 +3,44 @@ package sdkusage
 import (
 	"os"
 	"path/filepath"
+	"time"
 
 	"github.com/grafana/plugin-validator/pkg/analysis"
 	"github.com/grafana/plugin-validator/pkg/analysis/passes/nestedmetadata"
 	"github.com/grafana/plugin-validator/pkg/analysis/passes/sourcecode"
+	"github.com/grafana/plugin-validator/pkg/githubutils"
+	"github.com/grafana/plugin-validator/pkg/logme"
 	"golang.org/x/mod/modfile"
 )
 
 var (
-	goSdkNotUsed  = &analysis.Rule{Name: "go-sdk-not-used", Severity: analysis.Error}
-	goModNotFound = &analysis.Rule{Name: "go-mod-not-found", Severity: analysis.Error}
+	goSdkNotUsed            = &analysis.Rule{Name: "go-sdk-not-used", Severity: analysis.Error}
+	goModNotFound           = &analysis.Rule{Name: "go-mod-not-found", Severity: analysis.Error}
+	goModError              = &analysis.Rule{Name: "go-mod-error", Severity: analysis.Error}
+	goSdkOlderThanTwoMonths = &analysis.Rule{
+		Name:     "go-sdk-older-than-2-months",
+		Severity: analysis.Warning,
+	}
+	goSdkOlderThanFiveMonths = &analysis.Rule{
+		Name:     "go-sdk-older-than-5-months",
+		Severity: analysis.Error,
+	}
 )
 
+var twoMonths = 30 * 2
```

Comment:
you could work with `time.Duration` here if you want to:

```go
var day, _ = time.ParseDuration("24h")
var twoMonths = day * 30 * 2
```

Not necessarily better since this is simple enough

Reply:
I'll see if maybe this allows "2 months" as input otherwise I'll leave it as is.